index.js

```diff
@@ -75,8 +82,41 @@ module.exports = function (source, map) {
     }
   });
 
-  function relativePath(fullPath) {
-    return path.relative(rootPath, fullPath);
+  function getAliases (options) {
+    // TODO: Investigate if there is a way to leverage
+    // the Webpack API to get all the aliases
+    var resolveAliases = options.resolve && options.resolve.alias;
+    var resolveLoaderAliases = options.resolveLoader && options.resolveLoader.alias;
+    return Object.assign({}, resolveAliases || {}, resolveLoaderAliases || {});
```

Comment:
Sure, I have no problem with using lodash for it.
Yeah, adding Babel is something I want to do since I make the same mistake as well from time to time. If you'd rather add Babel than an `Object.assign` shim you'd be more than welcome!